core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -131,7 +131,7 @@ start_link(Name, ExpirePeriod, Props) ->
             gen_server:start_link({'local', Name}, ?MODULE, [Name, ExpirePeriod, Props], []);
         BindingProps ->
             lager:debug("started new cache process (gen_listener): ~s", [Name]),
-            Bindings = [{'conf', ['federate'|P]} || P <- BindingProps],
+            Bindings = build_bindings(BindingProps),
```

Comment:
@Dinkor try this here.
`Bindings = [{'conf', ['federate'|P]} || P <- build_bindings(BindingProps]),`